tests/materialize/materialize-databricks/cleanup.sh

```diff
@@ -10,6 +10,7 @@ function dropTable() {
 }
 
 # Remove materialized tables.
+dbsqlcli -e "SELECT 1"
```

Comment:
What's this for?

Reply:
@williamhbaker the first time we run `dbsqlcli`, it just "configures" dbsqlcli with the given ENV variables, so I've put this dummy `SELECT 1` query. This is necessary since in the CI we start with a fresh environment,  and dbsqlcli needs to be initialised before it will run the subsequent queries successfully. A bit of an odd design / behavior.